tests/integration/bond_test.py

```diff
@@ -227,19 +228,17 @@ def test_set_bond_mac_address(eth1_up):
         libnmstate.apply(bond_state)
 
         current_state = statelib.show_only(('bond99', 'eth1'))
-        bond99_cur_state = current_state[INTERFACES][0]
-        eth1_cur_state = current_state[INTERFACES][1]
-        assert bond99_cur_state[Interface.MAC] == MAC0.upper()
-        assert bond99_cur_state[Interface.MAC] == eth1_cur_state[Interface.MAC]
+        current_state_iface_macs = get_all_macs(current_state)
+        assert current_state_iface_macs[0] == MAC0.upper()
+        assert current_state_iface_macs[0] == current_state_iface_macs[1]
 
         bond_state[Interface.KEY][0][Interface.MAC] = MAC1
         libnmstate.apply(bond_state)
 
         current_state = statelib.show_only(('bond99', 'eth1'))
-        bond99_cur_state = current_state[INTERFACES][0]
-        eth1_cur_state = current_state[INTERFACES][1]
-        assert bond99_cur_state[Interface.MAC] == MAC1.upper()
-        assert bond99_cur_state[Interface.MAC] == eth1_cur_state[Interface.MAC]
+        current_state_iface_macs = get_all_macs(current_state)
+        assert current_state_iface_macs[0] == MAC1.upper()
```

Comment:
This can be one assert with all three members (or use the assert helper directly).